Example/Example Swift/ComponentAPICoordinator.swift

```diff
@@ -244,13 +245,21 @@ extension ComponentAPICoordinator {
         closeComponentAPI()
     }
     
-    fileprivate func push<T>(viewController: UIViewController, removingViewControllerOfType: T.Type) {
+    fileprivate func push<T: UIViewController>(viewController: UIViewController, removing viewControllers: [T?]) {
```

Comment:
It might be actually better to make it an extension to the `UINavigationController`. But just a suggestion.

Reply:
If it were for the Screen API, I would do it. But I don't see an advantage of creating it for the Component API.

Reply:
What's the difference?